src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -189,6 +189,7 @@ public String getQuestionResultStatisticsHtml(
             templateToUse = FormTemplates.RANK_RESULT_RECIPIENT_STATS;
         }
         DecimalFormat df = new DecimalFormat("#.##");
+        Map<String, Integer> recipientSelfRanks = generateSelfRankForEachRecipient(responses);
```

Comment:
I got a doubt about this. Why should we run this inside

``` java
if (shouldSelfBeExcludedFromRankings(question)) {

}
```

Isn't `shouldSelfBeExcludedFromRankings(question)` concerned only about the last column, which is `Average Rank Excluding Self`? Aren't we supposed to display `Self rank` regardless of value of `shouldSelfBeExcludedFromRankings(question)`? :thinking: